app/views/request_item_comments/_request_item_comment.html.erb

```diff
@@ -1,12 +1,11 @@
-<div class="container">
-  <div class="row">
+<div class="container word-wrap break-word">
+  <div class="row word-wrap break-word">
     <div class="col-sm-2">
```

Comment:
@duckgirl07 this only breaks the text if it has spaces... if you do a long word with no breaks then it overflows?